DFXR3Editor/Defs/DefActionID.xml

```diff
@@ -193,36 +325,36 @@
 				<field name="" dataType="f32"/>
 			</Fields2>
 		</ActionID>
-		<ActionID ID="602" name="Rectangle(Color)">
+		<ActionID ID="602" name="Basic Rectangle">
```

Comment:
The fact the rectangle is colored, is relatively important as there's rectangles that do all sort of things.
This is a non textured colored rectangle. That's what's special about it.